jans-config-api/shared/src/main/java/io/jans/configapi/core/util/Jackson.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Janssen Project software is available under the MIT License (2008). See http://opensource.org/licenses/MIT for full text.
+ *
+ * Copyright (c) 2020, Janssen Project
+ */
+
+package io.jans.configapi.core.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.AnnotationIntrospector;
+import com.fasterxml.jackson.databind.DeserializationConfig;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationConfig;
+import com.fasterxml.jackson.databind.introspect.JacksonAnnotationIntrospector;
+import com.github.fge.jackson.JacksonUtils;
+import com.github.fge.jsonpatch.JsonPatch;
+import com.github.fge.jsonpatch.JsonPatchException;
+import com.google.common.base.Preconditions;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * @author Yuriy Zabrovarnyy
+ */
+public class Jackson {
```

Comment:
Is it copy/paste of class from common module? Lets re-use code.
https://github.com/JanssenProject/jans/blob/main/jans-config-api/common/src/main/java/io/jans/configapi/util/Jackson.java

Reply:
Reusing the shared lib code